WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
 	
 	allowed_to_hold = {
 		is_adult = yes
 		is_ruler = no
 		is_female = no
-		trait = poet
+		OR = {
+			trait = poet
+			learning = 14
+		}
```

Comment:
It isn't at all. This isn't binary, I explained the alternative very clearly above, so I refer to that:

> There are already events for creating new ones in vanilla, and otherwise we can add a decision.